indigo/distribution.yaml

```diff
@@ -7270,6 +7270,11 @@ repositories:
       url: https://bitbucket.org/crl/multisense_ros
       version: default
     status: developed
+  muse_bldc_motor_drive:
+    doc:
+      type: git
+      url: https://bitbucket.org/muserobotics/muse_ros_package
```

Comment:
Yes, they've already cleared your account. Now the CI test is failing because of your syntax. Please fix the url as I mentioned above so that this will pass the tests.